modules/installation-vsphere-regions-zones.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+//* installing-vsphere-installer-provisioned-customizations.adoc [IPI]
+//* installing-vsphere-installer-provisioned-network-customizations.adoc [IPI]
+//* installing-vsphere.adoc [UPI]
+//* installing-vsphere-network-customizations.adoc [UPI]
+//* installing-restricted-networks-installer-provisioned-vsphere.adoc [IPI]
+//* installing-restricted-networks-vsphere.adoc [IPI]
+
+:_content-type: CONCEPT
+[id="installation-vsphere-regions-zones_{context}"]
+= VMware vSphere region and zone enablement
+
+You can deploy an {product-title} cluster to multiple vSphere datacenters, or regions, that run in a single VMware vCenter. Each datacenter can run multiple clusters, or zones. This configuration reduces the risk of a hardware failure or network outage causing your cluster to fail.
+
+You can also specify multiple regions and zones on a cluster after installation. You must upgrade your existing cluster to {product-title} {product-version} before you an configure a multiple region and zone environment.
+
+You must create an installation configuration file and then modify for the purposes of region and zone enablement, because deploying a cluster to a single vSphere datacenter is the default installation configuration.
+
+For {product-title} {product-version}, the default `install-config.yaml` file includes `vcenters` and `failureDomains` fields, so that you can choose to specify multiple datacenters, region, and zone information for your cluster. You can leave these fields blank if you want to install an {product-title} cluster in a vSphere environment that consists of single datacenter running in a VMware vCenter.
+
+The following list describes terms associated with defining zones and regions for your cluster:
+
+* Failure domain: Establishes the relationships between a region and zone. You define a failure domain by using vCenter objects, such as a `datastore` object. A failure domain defines the vCenter location for {product-title} cluster nodes.
+* Region: Specifies a vCenter datacenter. You define a region by using a tag from a created tag category, such as `openshift-region`. 
+* Zone: Specifies a vCenter cluster. You define a zone by using a tag from from a created tag category, such as `openshift-zone`. 
```

Comment:
Hi @jcpowermac . The tag category names here are just examples? A user could call the tags whatever they want?

Reply:
Tag categories are hard coded - they *must* be openshift-region and openshift-zone.

The *tags* can be anything

Reply:
Updated. I'm wondering about step 1 in [Configuring regions and zones for a VMware vCenter](https://55767--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere-installer-provisioned-network-customizations.html#configuring-vsphere-regions-zones_installing-vsphere-installer-provisioned-network-customizations): does a user have to explicitly name these exact tag categories? Of are the tag categories automatically created after the installation program creates the vanilla YAML config file?

Reply:
> does a user have to explicitly name these exact tag categories?
yes. 

The user must create two tag categories named: `openshift-region` and `openshift-zone`

The user must create tags that correspond to their arbitrary region/zone labels that is associated via the above tag categories (openshift-region,openshift-zone) that are attached to the defined Datacenter and Cluster in vCenter. 



> Of are the tag categories automatically created after the installation program creates the vanilla YAML config file?

no

Reply:
example
![image](https://user-images.githubusercontent.com/5401402/223183668-ed8158b7-8f20-4ac5-9c1d-672f7d1deb36.png)

Reply:
Thanks. I'll update the instructions to inform a user that they must specifically create and name these tag categories.

Reply:
Sorry to ask, @jcpowermac : What would happen if a user named the tag categories something else?

Reply:
Before install: the installation will fail
After install: :man_shrugging: I think nodes go offline